tests/test_cluster.py

```diff
@@ -2285,5 +2285,22 @@ def test_update(self):
         with self.assertRaises(SystemExit):
             qds.main()
 
+class TestClusterACL(QdsCliTestCase):
+    def test_get_object_policy(self):
+        sys.argv = ['qds.py', 'cluster', 'object_policy', 'get', '--cluster-id', '1001']
```

Comment:
why is `cluster` required in `--cluster-id` ? You've already specified `cluster` why do it twice ? Also I dont like the way the command line is setup. Maybe the recommendation is completely broken across the SDK. However, the preferred pattern is 

    qds.py object action [args]


With that pattern, it should be

    qds.py cluster get_object_policy --id